autosklearn/estimators.py

```diff
@@ -253,11 +253,11 @@ def fit(self, *args, **kwargs):
         self._automl = self.build_automl()
         super(AutoSklearnEstimator, self).fit(*args, **kwargs)
 
-    def fit_ensemble(self, task=None, metric=None, precision='32',
+    def fit_ensemble(self, y, task=None, metric=None, precision='32',
```

Comment:
Well, I'd personally even use the `X` parameter too, I'd say this signature would be even more convenient (for all the estimators). IMO the only negative side is the force to use some parameters that doesn't matter and do not override the data from `datamanager`.